src/bench/bench.cpp

```diff
@@ -88,7 +88,7 @@ bool benchmark::State::KeepRunning()
     lastCycles = nowCycles;
     ++count;
 
-    if (now - beginTime < maxElapsed) return true; // Keep going
+    if (now - beginTime < maxElapsed || count == 1) return true; // Keep going
```

Comment:
yes, my read it can't be 1 or 0, it must be >1 and then the decrements reduces it to 1.  If it is possible to be 0 at the division then I believe there is a bug elsewhere (in addition).